packages/dds/matrix/src/test/time/.mocharc.cjs

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Mocha configuration file to run memory-profiling tests
+ */
+
+module.exports = {
+	"exit": true,
+	"fgrep": ["@Benchmark", "@ExecutionTime"],
+	"node-option": ["expose-gc", "gc-global", "unhandled-rejections=strict"], // without leading "--"
+	"recursive": true,
+	"reporter": "@fluid-tools/benchmark/dist/MochaReporter.js",
+	"reporterOptions": ["reportDir=.memoryTestsOutput/"],
+	"require": ["node_modules/@fluid-internal/mocha-test-setup"],
+	"spec": ["dist/test/time/**/*.spec.*js", "--perfMode"],
+	"timeout": "90000",
```

Comment:
Let's start with a lower timeout, maybe 15000 and increase it if we need to? I'd try to go as low as we can.

Reply:
Updated